bpmsuite/bpmsuite70-businesscentral-monitoring-with-smartrouter.json

```diff
@@ -3,21 +3,35 @@
     "apiVersion": "v1",
     "metadata": {
         "annotations": {
-            "description": "Application template for Red Hat JBoss BPM Suite 7.0 Business Central Monitoring and MySQL applications.",
+            "description": "Application template for Red Hat JBoss BPM Suite 7.0 which includes only a Business Central.",
```

Comment:
Description should mention also Smart router.

Reply:
Will add that.